packages/pds/src/api/app/bsky/actor/createScene.ts

```diff
@@ -9,7 +9,7 @@ import AppContext from '../../../../context'
 
 export default function (server: Server, ctx: AppContext) {
   server.app.bsky.actor.createScene({
-    auth: ctx.accessVerifier,
+    auth: ctx.accessVerifierCheckTakedown,
```

Comment:
Oops, I think I responded to an old version of your comment!  All that still applies, but to answer more specifically—

I applied the takedown check on any routes that cause writes.  The idea is that a. writes are less common than reads, and we can afford the db roundtrip to verify write requests; and b. we want to _immediately_ disallow the user from being able to put up content, rather than the 15min grace period a fresh access token would give them.

Reply:
ah got it. yeah that makes sense to me 👍